### ISSUE TITLE
handling data versions in data manager

### ISSUE THREAD

**Issue by annagree** [Sage-Bionetworks/Agora#676]
- parameterize synid into json file and put json file into agora data manager
- move data version into the Agora data manager
- modify script so the Agora app reads the data version from data manager

@kdaily @zaro0508 please clarify the above as needed

**Comment by zaro0508** [comment]
I would like to add that since synapse entity id and version are a matched pair a full version description requires both information.   Therefore it would make more sense to show the data version on the agora web page as `<synapse entity id>-v<version number>` (i.e. `syn18387112-v10`)

**Comment by kdaily** [comment]
Good idea. Format should be `syn123.4` for compatibility elsewhere.

**Comment by Zaknarfen** [comment]
I made the first draft for the `agora-data-manager` [here](https://github.com/Sage-Bionetworks/agora-data-manager/pull/7). The changes to the Agora app itself will come as soon as @kdaily updates the data. I've already set the `data-version` to `11` there.

One thing to notice, if we are using the Synapse id for the version we need to get it from somewhere when the app itself is running. I'll change the `package.json` in Agora from `data-version` to `data-file` and put the Synapse id there so I can read later on and build the data version as @zaro0508 proposed.
